README.md

```diff
@@ -8,206 +8,18 @@ This repo requires BOTH a US ROM and a JP ROM in order to build. Place baserom.u
 
 This repo needs gcc in order to be able to build it. To install it, run `sudo apt install gcc-mips-linux-gnu`
```

Comment:
Please link to wiki https://github.com/HackerN64/HackerSM64/wiki/Installing-HackerSM64 for installation guide.